src/context/schemas/message.schema.json

```
@@ -0,0 +1,55 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://fdc3.finos.org/schemas/message.schema.json",
+  "type": "object",
+  "title": "Message",
+  "properties": {
```

```suggestion
  "allOf": [{ "$ref": "context.schema.json#" }],
  "properties": {
```
missing "allOf": [{ "$ref": "context.schema.json#" }], above porperties to import the context schema and extend it